scripts/extensions/sams/src/components/assets/assetListItem.tsx

```diff
@@ -46,16 +49,26 @@ export class AssetListItem extends React.PureComponent<IProps> {
     stopClickPropagation(e: React.MouseEvent<HTMLDivElement>) {
         e.stopPropagation();
     }
+    onCheckboxClick(e: React.MouseEvent<HTMLDivElement>) {
+        this.stopClickPropagation(e);
+        this.props.updateSelectedAssetIds(this.props.asset);
+    }
 
     render() {
         const {gettext, longFormatDateTime} = superdeskApi.localization;
         const actions = getDropdownItemsForActions(this.props.asset, this.props.actions);
 
         return (
-            <ListItem onClick={this.onItemClick} selected={this.props.selected} shadow={1}>
+            <ListItem onClick={this.onItemClick} selected={this.props.selected || this.props.itemSelected} shadow={1}>
                 <ListItemBorder />
                 <ListItemColumn>
                     <Icon name={getIconTypeFromMimetype(this.props.asset.mimetype)} />
+                    <div className="sd-grid-item__checkbox" onClick={this.onCheckboxClick}>
```

Comment:
There is a specific set of classes to use with the `ListItem` checkboxes. See the Planning playground for examples
https://ui-framework.superdesk.org/#/playgrounds/planning

Reply:
I used the first example in Planning playground.

Reply:
By design the UI Framework uses BEM for naming convention and to enforce usage. In this case, classes starting with `sd-grid-item__` should only be used with a parent of `sd-grid-item`.
https://ui-framework.superdesk.org/#/components/list-item shows how to use `sd-list-item` and at the bottom of that page you will see an example of the checkbox on a list item